### PR TITLE
Bratseth/test type checking

### PR DESCRIPTION
**Pull request by bratseth**

### CODE REVIEW

File: searchlib/src/main/java/com/yahoo/searchlib/rankingexpression/evaluation/FeatureNames.java [hunk 1]
```diff
@@ -0,0 +1,76 @@
+package com.yahoo.searchlib.rankingexpression.evaluation;// Copyright 2018 Yahoo Holdings. Licensed under the terms of the Apache 2.0 license. See LICENSE in the project root.
```
Comment (lesters) [inline searchlib/src/main/java/com/yahoo/searchlib/rankingexpression/evaluation/FeatureNames.java]: Well, I guess this does count as having the license header on the first line....

File: searchlib/src/test/java/com/yahoo/searchlib/rankingexpression/evaluation/FeatureNamesTestCase.java [hunk 2]
```diff
@@ -0,0 +1,48 @@
+package com.yahoo.searchlib.rankingexpression.evaluation;// Copyright 2018 Yahoo Holdings. Licensed under the terms of the Apache 2.0 license. See LICENSE in the project root.
```
Comment (lesters) [inline searchlib/src/test/java/com/yahoo/searchlib/rankingexpression/evaluation/FeatureNamesTestCase.java]: Well, I guess this also counts as having the license header on the first line....

**Review by lesters — APPROVED** [review]
LGTM. Check license headers. 👍

**Comment by bratseth** [comment]
IntelliJ ... thanks, fixed.

**Pull request merged**